lametro/templates/partials/board_report_statuses.html

```diff
@@ -0,0 +1,33 @@
+<table class="table">
+    <thead>
+        <tr>
+            <th>Status</th>
+            <th class="desktop-only">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for status, obj in BILL_STATUS_DESCRIPTIONS.items %}
+        <tr>
+            <td>
+                <p class='nowrap'>
+                    {% if obj.search_term != None %}
+                    <a href='/search/?selected_facets=bill_type_exact%3A{{obj.search_term}}'>{{status}}</a>
+                    {% else %}
+                    <p>{{status}}</p>
```

Comment:
What do you think about title casing this for display? I believe there's a Django template filter for that.